src/main/java/salesforce/entities/Opportunity.java

```diff
@@ -0,0 +1,285 @@
+
+    /**
+     * Variable for the type.
+     */
+    private String type;
+
+    /**
+     * Variable for the amount.
+     */
+    private String amount;
+
+    /**
+     * Variable for the probability.
+     */
+    private String probability;
+
+    /**
+     * Variable for the description.
+     */
+    private String description;
+
+    /**
+     * Variable for the closeDate.
+     */
+    private String closeDate;
+
+    /**
+     * Variable for the nextStep.
+     */
+    private String nextStep;
+
+    /**
+     * Constant for the Days.
+     */
+    private static final int DAYS = 3;
+
+    /**
+     * Constant for the name.
+     */
+    private static final String OPPORTUNITY_NAME = "name";
+
+    /**
+     * Constant for the type.
+     */
+    private static final String OPPORTUNITY_TYPE = "type";
+
+    /**
+     * Constant for the amount.
+     */
+    private static final String OPPORTUNITY_AMOUNT = "amount";
+
+    /**
+     * Constant for the probability.
+     */
+    private static final String OPPORTUNITY_PROBABILITY = "probability";
+
+    /**
+     * Constant for the description.
+     */
+    private static final String OPPORTUNITY_DESCRIPTION = "description";
+
+    /**
+     * Constant for the closeDate.
+     */
+    private static final String OPPORTUNITY_CLOSE_DATE = "closeDate";
+
+    /**
+     * Constant for the nextStep.
+     */
+    private static final String OPPORTUNITY_NEXT_STEP = "";
+
+    /**
+     * Variable of date.
+     */
+    private Date todayDate = new Date();
+
+    /**
+     * Returns the id of the Opportunity.
+     *
+     * @return the id as string.
+     */
+    public String getId() {
+        return id;
+    }
+
+    /**
+     * Sets the id of Opportunity sending a string.
+     *
+     * @param id for the Opportunity.
+     */
+    public void setId(final String id) {
+        this.id = id;
+    }
+
+    /**
+     * Returns the name of the Opportunity.
+     *
+     * @return the name as string.
+     */
+    public String getName() {
+        return name;
+    }
+
+    /**
+     * Sets the name of Opportunity sending a string.
+     *
+     * @param name for the Opportunity.
+     */
+    public void setName(final String name) {
+        this.name = name;
+    }
+
+    /**
+     * Returns the type of the Opportunity.
+     *
+     * @return the type as string.
+     */
+    public String getType() {
+        return type;
+    }
+
+    /**
+     * Sets the id of Opportunity sending a string.
+     *
+     * @param type for the Opportunity.
+     */
+    public void setType(final String type) {
+        this.type = type;
+    }
+
+    /**
+     * Returns the amount of the Opportunity.
+     *
+     * @return the amount as string.
+     */
+    public String getAmount() {
+        return amount;
+    }
+
+    /**
+     * Sets the id of Opportunity sending a string.
+     *
+     * @param amount for the Opportunity.
+     */
+    public void setAmount(final String amount) {
```

Comment:
Why your variable name is "amount"? the variable name should be more descriptive

Reply:
In this line "* @param amount for the Opportunity." I describe the variable